docs/FeatureManagement/Clients/SpringCloud/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
File name should use "FeatureManagement" instead of "FeatureFlag" since this is the schema for feature management, not an individual feature flag.